woocommerce-gateway-ebanx/includes/class-wc-ebanx-global-gateway.php

```diff
@@ -282,18 +305,36 @@ public function init_form_fields()
 				'type'  => 'title',
 				'class' => 'ebanx-payments-option'
 			),
-			'due_date_days'             => array(
-				'title'   => __('Days to Expiration', 'woocommerce-gateway-ebanx'),
-				'type'    => 'select',
-				'class'   => 'ebanx-select ebanx-payments-option',
-				'options' => array(
-					'1' => '1',
-					'2' => '2',
-					'3' => '3',
+		));
+
+		if (in_array($this->merchant_currency, WC_EBANX_Gateway_Utils::$LOCAL_CURRENCIES) ) {
```

Comment:
I think we can improve this... You're using if...else for this, we can create just one array and change only some properties of the array, because we're duplicating the `title, class, description, desc_tip` properties. Got it?